torch/csrc/jit/ir/constants.cpp

```diff
@@ -70,7 +70,7 @@ c10::optional<Value*> tryInsertConstant(
   Node* n = g.create(prim::Constant);
   if (val.isTensor()) {
     at::Tensor ref = val.toTensor();
-    if (!ref.defined()) {
+    if (!ref.defined() || !ref.has_storage()) {
```

Comment:
This isn't disabling inlining it, it's replacing it with a None constant, which is probably not what we want

Reply:
I did not phrase it properly in my comment.  We need to preserve the attribute so setstate/getstate works properly.